website/content/docs/secrets/mongodbatlas.mdx

```diff
@@ -2,49 +2,45 @@
 layout: docs
 page_title: MongoDB Atlas - Secrets Engines
 description: |-
-  The MongoDB Atlas Secrets Engine for Vault generates MongoDB Atlas
+  The MongoDB Atlas secrets engine for Vault generates MongoDB Atlas
   Programmatic API Keys dynamically.
 ---
 
 # MongoDB atlas secrets engine
 
-The MongoDB Atlas Secrets Engine generates Programmatic API keys. The created MongoDB Atlas secrets are
+The MongoDB Atlas secrets engine generates Programmatic API keys. The created MongoDB Atlas secrets are
 time-based and are automatically revoked when the Vault lease expires, unless renewed.
 
 Vault will create a Programmatic API key for each lease that provide appropriate access to the defined MongoDB Atlas
 project or organization with appropriate role(s). The MongoDB Atlas Programmatic API Key Public and
 Private Keys are returned to the caller. To learn more about Programmatic API Keys visit the
-[Programmatic API Keys Doc](https://docs.atlas.mongodb.com/reference/api-docs/apiKeys/).
+[Programmatic API Keys Doc](https://docs.atlas.mongodb.com/configure-api-access/#programmatic-api-keys).
 
-  <Note>
-
-  The information below relates to the **MongoDB Altas secrets engine**. Refer to the
-  [MongoDB Atlas **database** secrets engine](/vault/docs/secrets/databases/mongodbatlas)
-  for information about using the MongoDB Atlas database plugin for the Vault
-  database secrets engine.
-
-  </Note>
+~> **NOTE: The information below relates to the **MongoDB Altas secrets engine**. Refer to the
```

Comment:
I thought the `~> ` style is considered legacy now? https://github.com/hashicorp/vault/blob/fd96ac79b922aed4d9e25743e2a4cc05ba6fca81/website/README.md#L107

Reply:
I don't know, is it? Is the Note tag the preferred way now?

Reply:
I went ahead and changed them all to use the Note tag. Let me know if there is another preferred format.